openedx/core/djangoapps/discussions/serializers.py

```diff
@@ -219,12 +209,15 @@ def to_representation(self, instance: DiscussionsConfiguration) -> dict:
         """
         course_key = instance.context_key
         payload = super().to_representation(instance)
-
+        course_pii_sharing_allowed = get_lti_pii_sharing_state_for_course(course_key)
         lti_configuration = LtiSerializer(
-            instance.lti_configuration,
-            context={'pii_sharing_allowed': get_lti_pii_sharing_state_for_course(course_key)}
+            instance=instance.lti_configuration,
+            context={'pii_sharing_allowed': course_pii_sharing_allowed}
```

Comment:
Do we need to pass 'pii_sharing_allowed' in the context now? Is it being used somewhere else except the to_representation method?

Reply:
good point. 
yes, it is used in the `update` method as well but I am quite certain that we can remove this from `get` calls.